src/distilabel/pipeline/base.py

```diff
@@ -122,6 +124,17 @@ def get_pipeline(cls) -> Union["BasePipeline", None]:
 _STEP_NOT_LOADED_CODE = -999
 
 
+def generate_random_name(num_chars: int = 8) -> str:
+    """Helper function to generate a random name for a pipeline by default.
+
+    Returns:
+        Random name for a pipeline.
+    """
+    return "pipeline_" + "".join(
+        random.choice(string.ascii_letters + string.digits) for _ in range(num_chars)
```

Comment:
why not use something like UUID?

Reply:
no real reason, I've just updated it as it's less code to maintain, thanks 😄

Reply:
Exactly my thoughts :)